src/components/controls/mouse-controls.js

```diff
@@ -0,0 +1,157 @@
+var registerControls = require('../../core/controls').registerControls;
+var THREE = require('../../lib/three');
+var debug = require('../../utils/debug');
+
+var warn = debug('components:mouse-controls:warn');
+
+/**
+ * Mouse-controls component.
+ *
+ * Updates rotation with mousedrag or pointerlock input.
+ */
+module.exports.Component = registerControls('mouse-controls', {
+  schema: {
+    enabled: { default: true },
+    pointerlockEnabled: { default: false },
```

Comment:
I'm always happy to get rid of things from the core :). The things that makes me happiest in the world is to remove code.